hack/install.sh

```diff
@@ -119,6 +129,24 @@ do_install() {
 	esac
 
 	if command_exists docker; then
+		version="$(docker -v | awk '{ print $3 }' | sed 's/\,//g')"
```

Comment:
but we pipe to sh not bash right, or is this ok

On Tue, Jan 19, 2016 at 2:39 PM, Tianon Gravi <notifications@github.com>
wrote:

> In hack/install.sh
> <https://github.com/docker/docker/pull/19349#discussion_r50187518>:
>
> > @@ -119,6 +129,24 @@ do_install() {
> >  	esac
> >
> >  	if command_exists docker; then
> > +		version="$(docker -v | awk '{ print $3 }' | sed 's/\,//g')"
>
> Add sub(/,$/, "", $3); before print in your awk instruction and you can
> drop sed here too, or just use -F '[ ,]+' so that comma becomes a
> separator, ala: docker -v | awk -F '[ ,]+' '{ print $3 }'
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/19349/files#r50187518>.
>